src/_includes/card.html

```diff
@@ -0,0 +1,33 @@
+{% for item in links.items %}
+<li class="card">
+  <div class="card-content">
+       <h2 class="card-title"> <a href="{{ item.url }}"> <img class="card-image" src="{{ item.image }}" alt="{{ item.alt}}"></a>
+      <a href="{{ item.url }}">  {{ item.title }} </a>         
+    </h2>
```

Comment:
Since the link wraps around the image and the text, you can simplify this to be:
```suggestion
    <h2 class="card-title">
      <a href="{{ item.url }}">
        <img class="card-image" src="{{ item.image }}" alt="{{ item.alt}}">
        {{ item.title }}
      </a>
    </h2>
```